app/build.gradle.kts

```diff
@@ -10,6 +12,7 @@ plugins {
     alias(libs.plugins.compose.compiler)
     alias(libs.plugins.serialization)
     alias(libs.plugins.screenshot)
+    id("org.jlleitschuh.gradle.ktlint") version "11.5.1"
```

Comment:
Please move the plugin declaration into your version catalog. Instead of hardcoding the version in your build file, you’d define it centrally in your `libs.versions.toml` which is already available.